README.md

```diff
@@ -23,21 +23,24 @@
 
 ## Packages
 
-| [Core](Resources/Core) | [UI](Resources/UI) | [Networking](Resources/Networking) | [PromiseKit](Resources/PromiseKit) | [Auth](Resources)
-| :-: | :-: | :-: | :-: |
+| [Core](Resources/Core) | [UI](Resources/UI) | [Networking](Resources/Networking) | [PromiseKit](Resources/PromiseKit) | [Auth](Resources/Auth)
+| :-: | :-: | :-: | :-: | :-: |
 
 ## Installation
 
 ### Swift Package Manager
 - In Xcode, click `File` -> `Add Packages...`  
-- Insert `https://github.com/poviolabs/PovioKit` in the search field.
-- Select `Dependency Rule` "Up to Next Major Version" with "2.0.0"
-
-Currently, there are three packages to choose from:
-- *PovioKit* (core)
-- *PovioKitNetworking* (networking library, depends on `core` and `promise` package)
-- *PovioKitPromise* (lightweight promises library)
-- *PovioKitUI* (UI components)
+- Insert `https://github.com/poviolabs/PovioKit` in the Search field.
+- Select a desired `Dependency Rule`. Usually "Up to Next Major Version" with "2.0.0".
```

Comment:
Should this be "2.1.0"?

Reply:
Not necessarily. Anything equal to `2.0.0` or above will update up to version 3.0.0 anyway.